### ISSUE TITLE
"Select the folder that Notational Velocity should use for reading and storing notes"

### ISSUE THREAD

**Issue by LearninStuff** [scrod/nv#332]
Every time I open Notational Velocity, I now get a dialog box to confirm which folder I am using for my Notational Data. This didn't used to happen. I did recently change my storage options from single database to plain text files - not sure if that was related timewise.
I am on Mountain Lion, NV version 2.0Beta5(8)
Is there a way for NV to remember my folder between uses?

**Comment by Archibald57** [comment]
Hi everyone,

Could anyone give me a tip here ? I recently ventured into this wonderful note-taking app, i.e. notational velocity, and worked for around two hours compiling, archiving old mac notes that I had on my computer.

Today, I changed directory (in the preferences panes), and "moved" it to my personal files, and it seems to have erased all my previous notes. As the option just specifies saying : "Read the notes in directory...", I thought it would let the "default" directory (of the mac Library) intact.

Do you think that I have any chance to recover these previous notes ? If it's not(e) the case, I find it pretty ill-fated and misleadind (I should have made a copy of the repository previously, but I forgot).

Thank you,

J.

**Comment by scrod** [comment]
Changing your notes directory does **not** move your notes, and it most certainly doesn't erase them. It simply does what the command says: it sets the folder from which your notes are **read**.

Your old notes remain in their default location within your user Library folder. You can read "Where are my notes, and how can I move them?" in the [General FAQ](https://github.com/scrod/nv/wiki/FAQ:-General-Use) if your goal is actually to move (or even find) your notes.

**Comment by Archibald57** [comment]
Here is my occasion to pay a special and warm thank to you : I could solve my problem, and I'm really impressed by the reactivity and efficiency of this software (and his curators) ; it's miles away from all that commercial sh** which does nothing but impeding you getting a stable and usable environnment of work and creation.

Cordially,

J.

**Comment by Archibald57** [comment]
Just another last question :
I set the saving mode in notational velocity on 'txt' files (in the preferences pane) ; I'm just anguished that the formatting of my notes might get lost when I close the software. I see that my notes remain in bold, italic, u.s.w., in notationnal velocity, while .txt files do not include these functionnalities. How come it keeps these data, and where ?

I hope I made myself clear enough,

I thank you in advance,

J.
